ROMFS/px4fmu_common/init.d-posix/airframes/10020_if750a

```diff
@@ -16,4 +16,34 @@ param set-default MNT_MODE_IN 4
 param set-default MNT_MODE_OUT 2
 param set-default MNT_DO_STAB 2
 
-set MIXER quad_x
+
+# PX4 default motor ordering:
+#  3     1
+#     ^
+#  2     4
+param set-default SYS_CTRL_ALLOC 1
+param set-default CA_AIRFRAME 0
+param set-default CA_ROTOR_COUNT 4
+
+# front right (CCW)
+param set-default CA_ROTOR0_PX 0.707107
```

Comment:
In case we know the actual dimensions of the vehicle we can fill them in but in case we don't I would just put 1,1 -1,1 -1,-1 1,-1 for the positions.

Reply:
Agree